gazelle/modules_mapping/builder.py

```diff
@@ -0,0 +1,70 @@
+import argparse
+import multiprocessing
+import subprocess
+import sys
+from datetime import datetime
+
+mutex = multiprocessing.Lock()
+
+
+def build(wheel):
+    print("{}: building {}".format(datetime.now(), wheel), file=sys.stderr)
+    process = subprocess.run(
+        [sys.executable, "-m", "build", "--wheel", "--no-isolation"], cwd=wheel
```

Comment:
Is assuming that .tar.gz packages have a build module a packaging standard? Do some need setup.py run?

Reply:
I'm not sure I understand this question. The build module here is provided by https://packaging.python.org/key_projects/#build, which implements https://www.python.org/dev/peps/pep-0517/.

After it builds the wheel from the source, we dig the file with `generator.py` in this same Python package.
Is your question around needing to run `setup.py` instead of looking at the packaged wheel only?